tags-config-service-impl/src/main/java/org/hypertrace/tag/config/service/ConfigServiceCoordinatorImpl.java

```diff
@@ -0,0 +1,113 @@
+package org.hypertrace.tag.config.service;
+
+import static org.hypertrace.tag.config.service.TagConfigConstants.TAG_CONFIG;
+import static org.hypertrace.tag.config.service.TagConfigConstants.TAG_RESOURCE_NAMESPACE;
+
+import com.google.protobuf.Value;
+import io.grpc.Channel;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.SneakyThrows;
+import org.hypertrace.config.proto.converter.ConfigProtoConverter;
+import org.hypertrace.config.service.v1.ConfigServiceGrpc;
+import org.hypertrace.config.service.v1.ConfigServiceGrpc.ConfigServiceBlockingStub;
+import org.hypertrace.config.service.v1.DeleteConfigRequest;
+import org.hypertrace.config.service.v1.DeleteConfigResponse;
+import org.hypertrace.config.service.v1.GetAllConfigsRequest;
+import org.hypertrace.config.service.v1.GetAllConfigsResponse;
+import org.hypertrace.config.service.v1.GetConfigRequest;
+import org.hypertrace.config.service.v1.GetConfigResponse;
+import org.hypertrace.config.service.v1.UpsertConfigRequest;
+import org.hypertrace.config.service.v1.UpsertConfigResponse;
+import org.hypertrace.core.grpcutils.client.RequestContextClientCallCredsProviderFactory;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+import org.hypertrace.tag.config.service.v1.Tag;
+
+public class ConfigServiceCoordinatorImpl implements ConfigServiceCoordinator {
```

Comment:
Shall this be final? I don't think we need to extend this class.

Reply:
Will make it final.